lib.ts

```diff
@@ -218,6 +218,10 @@ export async function getDestination(platform: string) {
       return ['-destination', `id=${id}`]
     case 'macOS':
       return ['-destination', 'platform=macOS']
+    case 'mac-catalyst':
+      return ['-destination', "platform=macOS,variant='Mac Catalyst'"]
+    case 'driver-kit':
+      return ['-destination', "platform=macOS,variant='DriverKit'"]
```

Comment:
I tried building CombineCloudKit for `mac-catalyst` with `swift: ~5.5` and `runs-on: macos-11`:
https://github.com/chris-araman/CombineCloudKit/pull/4/checks?check_run_id=2866696038

Looks like we need to use quote the arguments differently:
```
'platform=macOS,variant=Mac Catalyst'
'platform=macOS,variant=DriverKit'
```

I verified the quote changes (before and after) on my Mac, though not through GHA.

Reply:
I adjusted the quoting

Reply:
Thanks, @mxcl. I'd do it myself, but I don't have push access to this repo.

I must be overlooking something. 🤔 I'm still seeing [this](https://github.com/chris-araman/CombineCloudKit/runs/2870232843?check_suite_focus=true):
```
/Applications/Xcode_13.0.app/Contents/Developer/usr/bin/xcodebuild -destination "platform=macOS,variant='Mac Catalyst'" -scheme CombineCloudKit-Package OTHER_SWIFT_FLAGS=-warnings-as-errors build
```

...even when explicitly using your commit:
```
Download action repository 'mxcl/xcodebuild@0fdf692470bb67ae91616eaf09f787cf0fbca3a9'
```

I don't understand where those quotes are coming from, but they're causing `xcodebuild` to fail. Any ideas?

Reply:
Can't see why. I looked into child_process.spawn which is used here and it doesn’t auto quote. So… weird. If this command line instantiation works without quotes then I'm flummoxed.

However perhaps we need to be more specific? Using the `id` perhaps like we do for iOS etc.

Reply:
Oh, I think I see the problem. 😉 

I'll see if I can figure out how to generate `index.js{.map}` on the fly.

Reply:
> However perhaps we need to be more specific? Using the id perhaps like we do for iOS etc.

I don't think this is necessary, as the only differences seem to be in `arch`. I don't think there's a way to build for a destination with a different Mac Catalyst runtime (v13/v14) than what is included in the host macOS. Maybe there will be in the future.